### PR TITLE
Option to set metrics/log API log levels

### PR DESCRIPTION
**Pull request by matrixik**

### CODE REVIEW

File: crowbar_framework/app/helpers/barclamp/monasca_helper.rb [hunk 1]
```diff
@@ -27,5 +27,21 @@ def log_levels(selected)
         selected.to_s
       )
     end
+
+    def api_log_levels(selected)
+      options_for_select(
+        [
+          ["CRITICAL", "CRITICAL"],
+          ["FATAL", "FATAL"],
```
Comment (samirjorina) [inline crowbar_framework/app/helpers/barclamp/monasca_helper.rb]: Setting **FATAL** results in error:
```
2017-04-18 16:01:44.570958 Traceback (most recent call last):
2017-04-18 16:01:44.570973   File "/srv/www/monasca-log-api/monasca-log-api.wsgi", line 5, in <module>
2017-04-18 16:01:44.571022     application = server.get_wsgi_app(config_base_path='/etc/monasca')
2017-04-18 16:01:44.571032   File "/usr/lib/python2.7/site-packages/monasca_log_api/server.py", line 106, in get_wsgi_app
2017-04-18 16:01:44.571852     global_conf=global_conf
2017-04-18 16:01:44.571869   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
2017-04-18 16:01:44.572027     return loadobj(APP, uri, name=name, **kw)
2017-04-18 16:01:44.572045   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
2017-04-18 16:01:44.572075     return context.create()
2017-04-18 16:01:44.572085   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
2017-04-18 16:01:44.572095     return self.object_type.invoke(self)
2017-04-18 16:01:44.572100   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 203, in invoke
2017-04-18 16:01:44.572107     app = context.app_context.create()
2017-04-18 16:01:44.572112   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
2017-04-18 16:01:44.572119     return self.object_type.invoke(self)
2017-04-18 16:01:44.572123   File "/usr/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
2017-04-18 16:01:44.572131     return fix_call(context.object, context.global_conf, **context.local_conf)
2017-04-18 16:01:44.572136   File "/usr/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
2017-04-18 16:01:44.572178     val = callable(*args, **kw)
2017-04-18 16:01:44.572185   File "/usr/lib/python2.7/site-packages/monasca_log_api/server.py", line 63, in launch
2017-04-18 16:01:44.572195     log.setup(CONF, 'monasca_log_api')
2017-04-18 16:01:44.572203   File "/usr/lib/python2.7/site-packages/oslo_log/log.py", line 267, in setup
2017-04-18 16:01:44.572316     _load_log_config(conf.log_config_append)
2017-04-18 16:01:44.572328   File "/usr/lib/python2.7/site-packages/oslo_log/log.py", line 229, in _load_log_config
2017-04-18 16:01:44.572351     raise LogConfigError(log_config_append, six.text_type(exc))
2017-04-18 16:01:44.572380 LogConfigError: Error loading logging config /etc/monasca/log-api-logging.conf: 'FATAL'

```
Please remove it from the list

File: crowbar_framework/app/helpers/barclamp/monasca_helper.rb [hunk 2]
```diff
@@ -27,5 +27,21 @@ def log_levels(selected)
...
+          ["ERROR", "ERROR"],
+          ["WARNING", "WARNING"],
+          ["INFO", "INFO"],
+          ["DEBUG", "DEBUG"],
+          ["NOTSET", "NOTSET"],
```
Comment (samirjorina) [inline crowbar_framework/app/helpers/barclamp/monasca_helper.rb]: I'd remove **NOTSET** level, as nobody (;) ) knows what it means. (Isn't it a kind of internal level?)

**Review by samirjorina — CHANGES_REQUESTED** [review]

File: crowbar_framework/app/helpers/barclamp/monasca_helper.rb (hunk 2 above)
Comment (matrixik) [reply crowbar_framework/app/helpers/barclamp/monasca_helper.rb]: Done

File: crowbar_framework/app/helpers/barclamp/monasca_helper.rb (hunk 1 above)
Comment (matrixik) [reply crowbar_framework/app/helpers/barclamp/monasca_helper.rb]: Removed FATAL log level because it's not supported by oslo_log

**Pull request closed without merging**